requirements.txt

```diff
@@ -1,14 +1,52 @@
+appdirs==1.4.4
```

Comment:
I ask again: Why are we adding a million requirements here? Why do we even need one new requirement?

Reply:
We basically need one package called **django-allauth** to implement Google OAuth2 in our django application. So, there are 10 requirements mentioned in main branch `requirements.txt`, adding one more will take this count to 11. Besides these 11 packages, the remaining are actually dependencies of those 11 packages.
I will now change the `requirements.txt` to mention only base packages so that it's not confusing anymore.